frontend/src/components/common/header/header.tsx

```diff
@@ -1,11 +1,13 @@
-import { useAppSelector, useDispatch, useVisible, useState } from 'hooks/hooks';
+import { useAppSelector, useDispatch, useState } from 'hooks/hooks';
 import { AppRoute, ButtonType } from 'common/enums/enums';
 import { RootState } from 'common/types/types';
 import { Button, Link, ImageWrapper } from 'components/common/common';
 import { auth as authActions } from 'store/actions';
 import logo from 'assets/img/logo.svg';
 import headerBell from 'assets/img/header-bell.svg';
 import { getAllowedClasses } from 'helpers/helpers';
+import Popup from 'reactjs-popup';
```

Comment:
we can't import it directly
we need to wrap it in the `common` components folder